modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java

```diff
@@ -1261,7 +1263,8 @@ public MenuItem getCheckinMenuItem() throws PortalException {
 	public boolean isCancelCheckoutActionAvailable() throws PortalException {
 		if ((_fileShortcut != null) ||
 			!_fileEntryDisplayContextHelper.
-				isCancelCheckoutDocumentActionAvailable()) {
+				isCancelCheckoutDocumentActionAvailable() ||
+			!_isAllowedFileVersionContentType()) {
```

Comment:
Adjectives go at the end in conditional sentences: _isFileVersionContentTypeAllowed().

Reply:
thanks!